Website/src/rendering/src/components/Vendors/VendorsGrid.tsx

```diff
@@ -10,8 +16,15 @@ type VendorsGridProps = ComponentProps & {
 };
 
 const VendorsGrid = (props: VendorsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasVendors = props.fields;
+
+  !hasVendors && console.log('Missing Datasource Item');
```

Comment:
Could all "Missing Datasource Item" `console.log` be converted to `console.warn` with datasource written as "Datasource" (preferred way) instead of "Data Source". In all modified components in this PR.

Using a `console.warn` would include the stack trace where the console message was added. This would give developers the name of the component that has no datasource.